test/unit/migrations/046-test.js

```diff
@@ -0,0 +1,52 @@
+import { strict as assert } from 'assert'
+import migration46 from '../../../app/scripts/migrations/046'
+
+describe('migration #46', function () {
+  it('should update the version metadata', async function () {
+    const oldStorage = {
+      'meta': {
+        'version': 45,
+      },
+      'data': {},
+    }
+
+    const newStorage = await migration46.migrate(oldStorage)
+    assert.deepEqual(newStorage.meta, {
+      'version': 46,
+    })
+  })
+
+  it('should delete ABTestController state', async function () {
+    const oldStorage = {
+      meta: {},
+      data: {
+        ABTestController: {
+          abTests: {
+            fullScreenVsPopup: 'control',
+          },
+        },
+        foo: 'bar',
+      },
+    }
+
+    const newStorage = await migration46.migrate(oldStorage)
+    assert.deepEqual(newStorage.data, {
+      foo: 'bar',
+    })
+  })
+
+  it('should do nothing if mkrMigrationReminderTimestamp state does not exist', async function () {
+    const oldStorage = {
+      meta: {},
+      data: {
+        AppStateController: {
+          bar: 'baz',
+        },
+        foo: 'bar',
+      },
+    }
```

Comment:
Is this old copypasta?

Reply:
Yes 😅 but technically it's valid

Reply:
Though the description is invalid 🤦 whoops